File: java-arquitectura-hexagonal/microservicio/dominio/src/main/java/com/ceiba/acuerdo/pago/modelo/dto/DtoAcuerdoPago.java
package com.ceiba.acuerdo.pago.modelo.dto;

import com.ceiba.acuerdopago.utilidades.enumeracion.EstadoAcuerdo;
import com.ceiba.cliente.modelo.entidad.Cliente;
import com.ceiba.deuda.modelo.entidad.Deuda;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;

import java.time.LocalDateTime;

@Getter
@Setter
@AllArgsConstructor
public class DtoAcuerdoPago {
	
	private Long idAcuerdoPago;
	private LocalDateTime fechaAcuerdo;
	private Double montoCuota;
	private Cliente cliente;
	private Deuda deuda;
	private EstadoAcuerdo estado;
	private Integer cantidadCuotas;
	private Long numeroReferencia;

}

File: angular-base/src/app/feature/acuerdo/shared/model/dto/DtoAcuerdoPago.ts
import { EstadoAcuerdo } from "../enumeraciones/estadoAcuerdo";

export class DtoAcuerdoPago {
    idAcuerdoPago: number;
    fechaAcuerdo;
    montoCuota: number;
    cliente: number;
    deuda: number;
    estado: EstadoAcuerdo;
    cantidadCuotas: number;
    numeroReferencia: number;

}